extension/src/experiments/columns/collect/util.ts

```diff
@@ -136,60 +120,34 @@ export const mergeAncestors = (
   }
 }
 
-const mergeValueIntoColumn = (column: Column, value: Value) => {
-  const valueType = getValueType(value)
-  if (!column.types) {
-    column.types = [valueType]
-  } else if (!column.types.includes(valueType)) {
-    column.types.push(valueType)
-  }
-
-  setStringLength(column, String(value).length)
-
-  if (valueType === 'number') {
-    mergeNumberIntoColumn(column, value as number)
-  }
-}
-
 const buildValueColumn = (
   path: string,
   parentPath: string,
   pathArray: string[],
   label: string,
   value: Value
 ) => {
-  const valueType = getValueType(value)
-
-  const newColumn: Column = {
+  return {
+    firstValueType: getValueType(value),
```

Comment:
[F] This is now called first value type because it only takes into account the first found value for a column